packages/clerk-js/src/core/clerk.ts

```diff
@@ -2010,8 +2006,9 @@ export class Clerk implements ClerkInterface {
    * The handshake payload is transported in the URL in development. In cases where FAPI is returning the handshake payload, but Clerk is being used in a client-only application,
    * we remove the handshake associated parameters as they are not necessary.
    */
-  #clearHandshakeFromUrl = () => {
+  #clearClerkQueryParams = () => {
     try {
+      removeClerkQueryParam(CLERK_SYNCED);
       removeClerkQueryParam('__clerk_handshake');
       removeClerkQueryParam('__clerk_help');
```

Comment:
Does it make sense to batch these ? To avoid calling replaceState multiple times ? Can this affect routers from host frameworks ?

Reply:
I don't think it makes a difference as we're calling replace here but I can definitely test